lib/OpenQA/Worker/Job.pm

```diff
@@ -814,6 +818,8 @@ sub _upload_results_step_0_prepare {
         $status{log} = $self->_log_snippet("$pool_directory/autoinst-log.txt", 'autoinst_log_offset');
         $status{serial_log} = $self->_log_snippet("$pool_directory/serial0", 'serial_log_offset');
         $status{serial_terminal} = $self->_log_snippet("$pool_directory/virtio_console.log", 'serial_terminal_offset');
+        $status{serial_terminal_user}
+          = $self->_log_snippet("$pool_directory/virtio_console_user.log", 'serial_terminal_offset');
```

Comment:
Since this is opening a different file, don't we need a distinct offset variable here?

Reply:
Thank you, you probably found a bug. I suppose this should work (I'll verify it tomorrow):

```
$status{serial_terminal_user}
     = $self->_log_snippet("$pool_directory/virtio_console_user.log", 'serial_terminal_user_offset');
```

Reply:
Obviously more changes were needed for `serial_terminal_user_offset`.

Reply:
@Martchus  I see fixed `virtio_console_user.log` in live console, thanks!
http://quasar.suse.cz/tests/2137#live (wicked, I'll test HPC, which actively uses it.)
But please verify.